docs/source/guides/contribution_guide.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+![NuCypher Unicorn](https://cdn-images-1.medium.com/max/800/1*J31AEMsTP6o_E5QOohn0Hw.png)
+
+
+## Running the Tests
+
+``` note:: A development installation including the solidity comppiler is required to run the tests
+```
+
+There are several test implementations in `nucypher`; However, the vast majority
+of test are written for execution with `pytest`.
+For more details see the [Pytest Documentation](https://docs.pytest.org/en/latest/)
+
+To run the tests:
+
+```bash
+(nucypher)$ pytest -s`
+```
+
+Optionally, to run the full, slow, verbose test suite run:
+
+```bash
+(nucypher)$ pytest --runslow -s
+```
+
+## Building Documentation
+
+``` note:: 'spinx', 'recommonmark', and 'sphinx_rtd_theme' are non-stantdard dependencies that be installed by running 'pip install -e .[docs]' from the project directory.
```

Comment:
```suggestion
``` note:: 'sphinx', 'recommonmark', and 'sphinx_rtd_theme' are non-standard dependencies that can be installed by running 'pip install -e .[docs]' from the project directory.
```